### DISCUSSION TITLE
Entering timesheets for users without access to Kimai

### DISCUSSION THREAD

**Discussion by artisleilands** [kimai/kimai#3257] [Questions & Answers]
Hi,

Situation: I would like to use Kimai to track employees' time. Some people work in the office, some in manufacturing where they don't have access to PC. I have to enter their timesheets from the past 2 years. Is entering timesheets one day at a time through "create timesheet" the only option for other users? Something like "weekly working hours" for other users would be very convenient, otherwise it is a nightmare. Am I missing something?
And subsequent requirement for a unique e-mail address is also not convenient in this case – it's good that e-mail activation is not mandatory, I can use non-existent addresses. That is sketchy, though. 
Possible solution: **Maybe it is possible to implement a "managed by" attribute, so that non-digitalized employees can be tracked as well?** I know it is possible now, but very difficult.

**Comment by kevinpapst** [comment]
There is a plugin that might help: https://www.kimai.org/store/mr-software-period-insert-bundle.html
It's not mine, so I can't answer more questions.

You can enter emails like user_abc@example.com - this is an official domain for testing purposes.
